lib/modules/apostrophe-pages/public/js/reorganize.js

```diff
@@ -345,21 +345,25 @@ apos.define('apostrophe-pages-reorganize', {
         }
       }
 
+      var changed = [];
+
       return async.series([
         move,
         trash
       ], function(err) {
         if (err) {
           self.errorOnMove();
         }
-        // Reflect changed slugs as needed
-        _.each(data.changed, function(info) {
-          var node = self.$tree.tree('getNodeById', info.id);
-          if (node) {
-            node.slug = info.slug;
-          }
-        });
         e.move_info.do_move();
+        if (trashing !== undefined) {
```

Comment:
@boutell I think we want this to run regardless of whether `trashing` is set. That gets us the right link in the tree right away when moving things out of the trash. Any concern about that?

Reply:
Otherwise the link, when `trashInSchema` is undefined, is prepended with `/trash`

Reply:
You are correct, pls change